### PR TITLE
fix typo in doc/project2-RaftKV.md

### PR DESCRIPTION
**Pull request by JLDataKu**
Fix typo in doc/project2-RaftKV.md.

### CODE REVIEW

File: doc/project2-RaftKV.md [hunk 1]
```diff
@@ -138,7 +138,7 @@ To save the hard state is also very easy, just update peer storage’s `RaftLoca
 
 ### Implement Raft ready process
 
-In project2 part A, you have built a tick-based Raft module. Now you need to write the outer process to drive it. Most of the code is already implemented under `kv/raftstore/peer_msg_handler.go` and `kv/raftstore/peer.go`.  So you need to learn the code and finish the logic of `proposeRaftCommand` and `HandleRaftReady`. Here are some interpretations of the framework.
+In project2 part A, you have built a tick-based Raft module. Now you need to write the outer process to drive it. Most of the code is already implemented under `kv/raftstore/peer_msg_handler.go` and `kv/raftstore/peer_storage.go`.  So you need to learn the code and finish the logic of `proposeRaftCommand` and `HandleRaftReady`. Here are some interpretations of the framework.
```
Comment (Connor1996) [inline doc/project2-RaftKV.md]: why

**Review by Connor1996 — COMMENTED** [review]
rest LGTM

File: doc/project2-RaftKV.md (hunk 1 above)
Comment (JLDataKu) [reply doc/project2-RaftKV.md]: Sorry, I misunderstood it.

Comment (Connor1996) [reply doc/project2-RaftKV.md]: so let's revert the change

**Review by Connor1996 — APPROVED** [review]
LGTM

**Review by ngaut — APPROVED** [review]

**Pull request merged**